src/components/TenantSelector/TenantSelector.stories.tsx

```diff
@@ -1,13 +1,18 @@
-/* eslint-disable react/no-multi-comp */
+// @ts-ignore
```

Comment:
Why?

Reply:
because storybook doesn't support typescript/tslint - so these will get ts-errors and the errortype is too large to ignore as a rule in total (any -reference)

Reply:
Ok fine